### ISSUE TITLE
How to access internal viewer object

### ISSUE THREAD

**Issue by IgelCampus** [tify-iiif-viewer/tify#163]
I am currently playing around with Tify and like it quite a lot. The homepage mentions (without an example) that the viewer object exposes the full [OpenSeadragon API](https://openseadragon.github.io/docs/OpenSeadragon.html). Could you provide an example how to access this? My naive attempt with 
```
let tify = new Tify ({
    container:   _el,
    manifestUrl: iiifUrl,
    viewer:      {
        showNavigator:      true
    }
});
let osd = tify.viewer; // !error: undefined
```
is not working.

**Comment by t11r** [comment]
The intended way is to [wait for the `ready` promise before accessing any API properties](https://github.com/tify-iiif-viewer/tify?tab=readme-ov-file#api), like this:

```js
await tify.ready
let osd = tify.viewer
```

Currently though, this is working for anything but `viewer`, for which you would have to wait manually. I’m adding an issue, this will get fixed.

**Comment by t11r** [comment]
Fixed in v0.31.0.
